packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx

```diff
@@ -12,19 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import _ from 'lodash';
+import _map from 'lodash/map';
```

Comment:
We can get rid of this `_map` submodule too, using the built-in JS function. Should I go ahead with that change?

Reply:
+1